src/components/auctions-table.tsx

```diff
@@ -48,8 +76,10 @@ function getAuctionStatus(
         )}
       </div>
     );
-  } else if (Number(blockNumber) > Number(auction.mintStartBlock)) {
-    return <>Closed</>;
+  } else if (Number(auction.bidStartBlock) < Number(blockNumber) && submitted) {
+    return <>{submitted} TXs pending...</>;
```

Comment:
I reckon we just write "Settling..." when `totalCount > 0`

Reply:
done!